project/Dependencies.scala

```diff
@@ -19,7 +19,7 @@ object Dependencies {
   val awsCloudWatch = "com.amazonaws" % "aws-java-sdk-cloudwatch" % awsClientVersion
   val membershipCommon = "com.gu" %% "membership-common" % "0.592"
   val scalaz = "org.scalaz" %% "scalaz-core" % "7.2.31"
-  val kinesis = "com.gu" % "kinesis-logback-appender" % "2.0.2"
+  val kinesis = "com.gu" % "kinesis-logback-appender" % "2.0.3"
```

Comment:
waiting for https://github.com/guardian/kinesis-logback-appender/pull/40 to ship

Reply:
this is out on maven central now https://mvnrepository.com/artifact/com.gu/kinesis-logback-appender